### PR TITLE
Update gui api

### PR DESCRIPTION
**Pull request by blacksponge**
## Purpose

Update the documentation following the introduction of language in URL.

## Context

https://github.com/zonemaster/zonemaster-gui/issues/377

## Changes

Add <lang> in all paths.

## How to test this PR

N/A

### CODE REVIEW

**Review by pnax — DISMISSED** [review]

File: docs/public/using/gui/api.md [hunk 1]
```diff
@@ -8,19 +8,21 @@ A path is added to the Zonemaster GUI base URL, e.g.
...
+The GUI will rewrite `<lang>/domain_check` to `<lang>/run-test` before the page
+is displayed. Note that `<lang>/domain_check/<domain>` is **not** rewritten and
+hence is an invalid path.
 
-The GUI will rewrite `test/<test-id>` to `result/<test-id>` before the result
-is displayed.
+The GUI will rewrite `<lang>/test/<test-id>` to `<lang>/result/<test-id>`
+before the result is displayed.
+
+The GUI will redirect from `<path>` to `<lang>/<path>` automatically.
```
Comment (matsduf) [inline docs/public/using/gui/api.md:28]: It should be specified how `<lang>` is selected for this case.

**Review by matsduf — CHANGES_REQUESTED** [review]
It looks fine besides a missing detail.

File: docs/public/using/gui/api.md (hunk 1 above)
Comment (blacksponge) [reply docs/public/using/gui/api.md]: I have clarified this section a bit more.

File: docs/public/using/gui/api.md [hunk 2]
```diff
@@ -8,19 +8,23 @@ A path is added to the Zonemaster GUI base URL, e.g.
...
+hence is an invalid path.
 
-The GUI will rewrite `test/<test-id>` to `result/<test-id>` before the result
-is displayed.
+The GUI will rewrite `<lang>/test/<test-id>` to `<lang>/result/<test-id>`
+before the result is displayed.
+
+The GUI will redirect from `<path>` to `<lang>/<path>` automatically using the
+user browser language, defaulting to the default language defined in the
+reverse proxy configuration.
```
Comment (matsduf) [inline docs/public/using/gui/api.md:30]: Does this mean
```
The GUI will redirect from `<path>` to `<lang>/<path>` automatically using the
user browser language, if that is defined for the installation. Else using the default 
language defined in the reverse proxy configuration.
```

Comment (blacksponge) [reply docs/public/using/gui/api.md:30]: I guess it does.

Comment (matsduf) [reply docs/public/using/gui/api.md:30]: Could you spell that out to make it easier to read?

Comment (blacksponge) [reply docs/public/using/gui/api.md]: I changed the wording.

**Pull request merged**